app/models.py

```diff
@@ -2130,7 +2129,7 @@ class CommunicationItem(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     va_profile_item_id = db.Column(db.Integer, nullable=False)
-    name = db.Column(db.Text(), nullable=False)
+    name = db.Column(db.String(), nullable=False)
```

Comment:
String/Text question again. Should not be unlimited length.

Reply:
added limited length